chart/epinio/templates/stage-scripts.yaml

```diff
@@ -95,7 +95,7 @@ data:
       exit 1
     fi
     find /workspace
-    for path in /workspace/source/env/* ; do export "$(basename "${path}")"="$(cat "${path}")" ; done
+    for path in $(ls /workspace/source/env/* 2>/dev/null) ; do export "$(basename "${path}")"="$(cat "${path}")" ; done
```

Comment:
Would `find` work here? Not a bash guru but maybe it nicer.

Reply:
`find` would also see the internal `..*` directories and links in the env directory. While constraining it should be possible, that also ups the complexity. IOW might not be as nice as you believe at the moment.